articles/neocities-learn.md

```diff
@@ -0,0 +1,24 @@
+---
+author: ele
```

Comment:
Use your full site name. Neowiki [links its contributors](https://minerobber.neocities.org/neowiki/writing-for-neowiki.html) and it cannot do this if you use a nickname.

Reply:
alright